### PR TITLE
Fix v1.0 bugs and update UG

### PR DESCRIPTION
**Pull request by zonglun99**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -264,20 +194,100 @@ __________________________________________
...
+    3. frozen duck | Qty: 1 | 10 Oct 2022
+__________________________________________
+```
+
+### Find items stored in Fridget: `find`
+Find all items associated with the keyword.
+
+Format: find <KEYWORD>
+
+* The `KEYWORD` should be something associated with the ingredient.
```
Comment (alvynben) [inline docs/UserGuide.md:241]: Can we reword this to:
* The `KEYWORD` should be the same as the name of the item you are looking for. 
* The `KEYWORD` can also be a part of the name of the item.

File: src/main/java/seedu/ui/Ui.java [hunk 2]
```diff
@@ -209,8 +209,8 @@ public String sortTypeMessage(String sortType) {
      */
     public void printListOfMatchingIngredients(ArrayList<Ingredient> listOfIngredients) {
         if (listOfIngredients.isEmpty()) {
-            String noMatchingTasks = "No matching tasks found!";
-            printLine(noMatchingTasks);
+            String noMatchingIngredient = "No matching ingredient found!";
+            printLine(noMatchingIngredient);
```
Comment (alvynben) [inline src/main/java/seedu/ui/Ui.java:213]: Thanks

**Review by alvynben — APPROVED** [review]

File: docs/UserGuide.md [hunk 3]
```diff
@@ -264,20 +194,101 @@ __________________________________________
...
 Format: notifs
 
+* `notifs`
+
 Example of usage:
 ````
 userInput: notifs
 __________________________________________
-Notification off!
+Turning notification off!
```
Comment (uosjapuelks) [inline docs/UserGuide.md:287]: Good to be past tense but I think this works too.

**Review by uosjapuelks — APPROVED** [review]
